platformio.ini

```diff
@@ -125,25 +140,24 @@ extra_scripts =
 	${env.extra_scripts}
 
 ; Build settings for the Raspberry Pico original
-[env:raspberrypico]
+[env:mobiflight_raspberrypico]
 platform = https://github.com/maxgerhardt/platform-raspberrypi.git
 board = pico
 framework = arduino
-board_build.core = earlephilhower			; select new core
-board_build.filesystem_size = 0M			; configure filesystem size. Default 0 Mbyte.
+board_build.core = earlephilhower						; select new core
+board_build.filesystem_size = 0M						; configure filesystem size. Default 0 Mbyte.
 lib_ldf_mode = chain+
-upload_protocol = mbed						; for debugging upoading can be changed to picoprobe
-;debug_tool = picoprobe						; and uncomment this for debugging w/ picoprobe
+upload_protocol = mbed									; for debugging upoading can be changed to picoprobe
+;debug_tool = picoprobe									; and uncomment this for debugging w/ picoprobe
 build_flags =
 	${env.build_flags}
-	-DUSE_INTERRUPT
+	-DMF_CUSTOMDEVICE_SUPPORT=0
+	'-DMOBIFLIGHT_TYPE="MobiFlight RaspiPico"'
 	-I./_Boards/RaspberryPi/Pico
-	-fpermissive
 build_src_filter =
 	${env.build_src_filter}
-	+<../_Boards/RaspberryPi>
 lib_deps =
 	${env.lib_deps}
 monitor_speed = 115200
 extra_scripts = 
-	${env.extra_scripts}
\ No newline at end of file
+	${env.extra_scripts}
```

Comment:
for all boards the environments are added with `mobiflight` to get the existent firmware file name (see above)